activerecord/test/cases/adapters/postgresql/hstore_test.rb

```diff
@@ -10,6 +10,12 @@ class Hstore < ActiveRecord::Base
       store_accessor :settings, :language, :timezone
     end
 
+    class FakeParameters
+      def to_h
```

Comment:
Once that's fixed this is fine to squash and merge. Don't forget to backport to 5-0-stable.